systemd/ceph

```diff
@@ -1,4 +1,4 @@
-#! /bin/bash
+#! /usr/bin/env bash
```

Comment:
@tchaikov 
I happy to say it is Linux only...
The amount of unexpected irritation I hear from my colleague makes me a content init user.